pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -207,14 +207,14 @@ func findOpenStackSubnet(client *gophercloud.ServiceClient, name, tag string) (s
 // Looks for a Neutron subnet by name, tag, network ID, CIDR and optionally
 // gateway IP, if it does not exist creates it. Will fail if two subnets match
 // all the criteria.
-func ensureOpenStackSubnet(client *gophercloud.ServiceClient, name, tag, netId, cidr string, gatewayIp *string) (string, error) {
+func ensureOpenStackSubnet(client *gophercloud.ServiceClient, name, tag, netId, cidr, gatewayIp string, allocationRanges []iputil.IPRange) (string, error) {
```

Comment:
It might be good to update the comment above to remove the optional for the gateway IP and state that a predefined allocationPools will be used in creation.

Reply:
Done.